docs/UserGuide.md

```diff
@@ -62,40 +61,71 @@ The GUI similar to the below should appear in a few seconds. Note how the app co
 
 ![Ui](images/Ui.png)
 
-5. Type the command where prompted "Enter command here" and press Enter to execute it. <br> Example: typing **`help`** and pressing Enter will open the help window which would bring provide you with a link and direct you to the [Command Summary](#command-summary).
+5. Type the command where prompted "Enter command here" and press Enter to execute it.
+    * Example: typing **`help`** and pressing Enter will open the help window which would bring provide you with a link and direct you to the [Command Summary](#command-summary).
 
 6. Refer to the [Features](#features) below for details of each command.
 
 ---
 
+## Command Summary
+### Understanding the command format
+1. Almost all the details of the commands are needed to be given before the command can be executed properly.
+    * **Compulsory** details are denoted by angled-brackets such as \<mandatory details\>.
+    * **Optional** details of the command that are enclosed in square brackets such as [optional details].
+        + Example: `command <recipe name or -x <recipe number from the list>> [optional details]`
+        + A clearer example of this can be seen in the [delete command](#delete-an-existing-recipe-delete), or [edit command](#edit-an-existing-recipe-edit).
+      
+2. What does it mean when the input command says "... separated by pipe"?
```

Comment:
instead of the word pipe, should it be just says"... seperated by |"? 
The next sentence will explain it